### DISCUSSION TITLE
Report: Black Mesa

### DISCUSSION THREAD

**Discussion by UnrealisticDino** [canonical/steam-snap#253] [Game Reports]
Both native and proton work, but proton has a bug where I have to click escape after a loading screen for it to finish loading otherwise it gets stuck.
```
os_release: 
    name:               "Ubuntu"
    version:            "22.04.2 LTS (Jammy Jellyfish)"
snap_info: 
    steam_revision:     154
    snapd_revision:     19361
lspci: 
    08:00.0:            NVIDIA Corporation GP104 [GeForce GTX 1070] (rev a1)
glxinfo: 
    gpu:                NVIDIA GeForce GTX 1070/PCIe/SSE2
    gpu_version:        4.6.0 NVIDIA 530.41.03
lscpu: 
    model_name:         AMD Ryzen 7 3800X 8-Core Processor
xdg_current_desktop:    ubuntu:GNOME
desktop_session:        ubuntu
```

**Comment by Nightmayr** [comment]
My experience (at least with Native) is different. I can successfully get into the game although there does seem to be quite a few visual glitches/artifacts on my setup (AMD Radeon RX580). I could possibly remedy this by playing with the graphics settings or using a different mesa driver version

Some of the artifacting:
![Screenshot from 2023-06-15 20-39-38](https://github.com/canonical/steam-snap/assets/24817147/cd0366fa-5106-4272-bc0c-58abcb410f61)

**Reply by UnrealisticDino** [reply]
After the steam update yesterday the game at least launches but crashes shortly after. Looking at protondb it seems like your experience is fairly normal though. Do you have the same issues with the deb version of steam?

**Reply by Nightmayr** [reply]
Just gave the deb a go as well and yeah the visual glitches are there as well so it's not a snap problem by the looks of it.

**Reply by UnrealisticDino** [reply]
Update. It's buggy but the game works if, and only if, I launch it in big picture mode.
